README.md

```diff
@@ -42,7 +42,10 @@ For notarization, you need the following things:
 1. Xcode 10 or later installed on your Mac.
 2. An [Apple Developer](https://developer.apple.com/) account.
 3. [An app-specific password for your ADC account’s Apple ID](https://support.apple.com/HT204397).
-
+4. Your app may need to be signed with harded-runtime and the following entitlements:
+    1. com.apple.security.cs.allow-jit
+    1. com.apple.security.cs.allow-unsigned-executable-memory
+    1. com.apple.security.cs.allow-dyld-environment-variables
```

Comment:
I think you only really need `com.apple.security.cs.allow-unsigned-executable-memory` for a vanilla Electron to function so would prefer to keep this list as minimal as possible, but possibly provide a link to a [list of other entitlements](https://developer.apple.com/documentation/bundleresources/entitlements) that specific apps might need.

Reply:
@Kilian `allow-jit` is definitely required if you want V8 to run it's optimizations 🤔

Reply:
Not sure about the dyld environment variables, seems app specific there

Reply:
seems this is subject to the app, how about we just mention it with the link of all the entitlement here?

Reply:
@Kilian kindly bump this PR, let me know your thoughts on this. 🙇

Reply:
(closing this because outdated)